apps/costcontrol/js/message_handler.js

```diff
@@ -42,7 +41,32 @@
   }
 
   // Close if in standalone mode
-  var closing, activity;
+  var closing;
+  var dataUsageActivity = {
```

Comment:
It does not make sense, make these variables global. Let's use locally. Furthermore, as said, it's better to rename the _activity/callback_ containers to some like `action`.